admin/app/adapters/scoring-configuration.js

```diff
@@ -0,0 +1,8 @@
+import ApplicationAdapter from './application';
+
+export default class ScoringConfigurationAdapter extends ApplicationAdapter {
+  updateCompetenceScoringConfiguration(newConfiguration) {
+    const url = `${this.host}/${this.namespace}/competence-for-scoring-configuration`;
+    return this.ajax(url, 'POST', { data: newConfiguration });
```

Comment:
On se perde un peu avec les 
`competence scoring configuration`
`scoring configuration`
`competence for scoring configuration` 
`configuration`
L'harmonization permettrait d'eviter l'adapter!?

Reply:
La page va pas rester comme dit dans les remarques de la PR, on a fait au plus rapide sans abstraction.